spec/models/footer_spec.rb

```diff
@@ -1,31 +1,26 @@
 require "rails_helper"
 
 RSpec.describe Footer, type: :model do
-  before {
-    allow(Rails).to receive(:root).and_return(file_fixture("_settings").join("..").cleanpath)
-  }
+  subject { described_class.load file_fixture("_settings/footer.yml").cleanpath }
 
   describe "#links" do
     it "reads urls correctly" do
-      footer = described_class.load file_fixture("_settings/footer.yml").cleanpath
-      expect(footer.links.map { |l| l.url }).to eql([
+      expect(subject.links.map { |l| l.url }).to eql([
         "/link-1",
         "/foo"
       ])
     end
 
     it "reads link text correctly" do
-      footer = described_class.load file_fixture("_settings/footer.yml").cleanpath
-      expect(footer.links.map { |l| l.text }).to eql([
+      expect(subject.links.map { |l| l.text }).to eql([
```

Comment:
another rubyism that doesn't need to be changed, but a shortcut when calling map with a single method on the object like this is `list.map(&:method_name_on_each_object)`, so here you could do `subject.links.map(&:text)`